docs/modules/migrate/pages/dmt-connection-config.adoc

```diff
@@ -0,0 +1,174 @@
+= Configure Connections
+:description: The migration cluster acts a client for both source and target clusters. You must configure the connections to allow communication between the migration cluster and both the source and target clusters. 
+
+{description}
+
+If the connections are not correctly configured, the migration fails.
+
+== Source Cluster Connection
+
+To complete the configuration, you need the following information from your source cluster:
+
+* The cluster name
+* The cluster member addresses
+* The data structures to migrate
+
+=== Update Cluster Information
+
+To update the cluster information, complete the following steps:
+
+. Navigate to the folder containing the extracted DMT package
+. Open the _migration_config/source/hazelcast-client.yaml_ file in your favorite editor
++
+NOTE: The _hazelcast-client.yaml_ file is a Hazelcast client configuration file, which can include any supported configuration.
++
+. Update the `cluster-name` field to match the name of your source cluster
+. Update the `cluster-members` field to match the addresses of the cluster members
+. Save the file
+
+=== Update Data Structures
+
+To update the data structure information, complete the following steps:
+
+. Navigate to the folder containing the extracted DMT package
+. Open the _migration_config/data/imap_names.txt_ and/or the _migration_config/data/replicated_map_names.txt_ file in your favorite editor
```

Comment:
also there are new files for cp structures. see https://github.com/hazelcast/hazelcast-mono/tree/0778d11cc407c3677c3a166308aecf5c01a5c418/distribution/src/root/migration_config/data

Reply:
I would just remove the file names from here as there are 5 of them now